pkgs/development/libraries/libdatrie/default.nix

```diff
@@ -7,6 +7,8 @@ stdenv.mkDerivation rec {
   pname = "libdatrie";
   version = "2019-12-20";
 
+  outputs = [ "bin" "out" "lib" "dev" ];
```

Comment:
Hmm I've never seen bin, out, and lib, before. Is out for `share`?

Reply:
There a few in nixpkgs already https://search.nix.gsc.io/?q=outputs%20%3D%20%5C%5B%20%22bin%22%20%22out%22&i=nope&files=&repos=